contributors/guide/README.md

```diff
@@ -0,0 +1,213 @@
+
+If you haven't done so already make sure you have found the correct SIG for your contribution. This will ensure faster responses and a streamlined code review. (see above)
+
+_Improvements needed_ 
+
+* the linked text is awfully similar to the section on [Code Review](https://github.com/kubernetes/community/blob/master/contributors/devel/community-expectations.md#code-review), linked above. Consolidate, and decide on _one place_ in this document where it fits best. It might even be important enough to include at top level in this document(no links).
+
+### Communication
+
+It is best to contact your SIG for issues related to the SIG's topic. Your SIG will be able to help you much more quickly than a general question would. Each SIG has a kubernetes slack channel that you can join as well.
+
+For questions and troubleshooting, please feel free to use any of the methods of communication listed [here](https://github.com/kubernetes/community/blob/master/communication.md). The [kubernetes website](https://kubernetes.io/community/) also lists this information.
+
+## GitHub workflow
+
+To check out code to work on, please refer to [this guide](https://github.com/kubernetes/community/blob/master/contributors/devel/development.md#workflow).
+
+_Improvements needed_ 
+
+* move github workflow into its own file in this folder.
+
+## Open a PR
+
+PR workflow is described [here](https://github.com/kubernetes/community/blob/master/contributors/devel/pull-requests.md#the-testing-and-merge-workflow).
+
+## Code Review
+
+_Improvements needed_
+* Clarify and streamline some of the explanations from [here](https://github.com/kubernetes/community/blob/master/contributors/devel/owners.md#code-review-process).
+
+* Explain OWNERS files and process (also found in above link)
+
+## Testing
+
+_Improvements needed_
+* link to testing process 
+* walkthrough of where to find what in the tests (how to use, how to debug)
+
+## Security
+
+_Improvements needed_
+* Please help write this section.
+
+## Documentation
+
+_Improvements needed_
+* Please help write this section.
+
+# Community
+
+If you haven't noticed by now, we have a large, lively, and friendly open-source community. We depend on new people becoming members and regular code contributors, so we would like you to come join us. To find out more about our community structure, different levels of membership and code contributors, please [explore here](https://github.com/kubernetes/community/blob/master/community-membership.md).
+
+_Improvements needed_
+
+* The top level k/community/README.md should be a good starting point for what the community is and does. (see above instructions on rewriting this file)
+
+## Events
+
+Kubernetes is the main focus of CloudNativeCon/KubeCon, held every spring in Europe and winter in North America. Information about these and other community events is available on the CNCF [events](https://www.cncf.io/events/) pages.
```

Comment:
Kubernetes is the main focus of CloudNativeCon/KubeCon, held twice per year in EMEA and in North America. Information about these and other community events is available on the CNCF [events](https://www.cncf.io/events/) pages.

Reply:
Edited. We don't necessarily run KubeCon only in winter and spring. Also, not necessarily Europe (but EMEA).